### ISSUE TITLE
Seeking censored German 1.0.6 build.

### ISSUE THREAD

**Issue by emoose** [emoose/re4-research#7]
Seems German owners previously had a censored game build, but Capcom eventually replaced it with the uncensored one, AFAIK anyone that bought it in Germany should still have access to the censored depot though (and seems some might still be stuck with that version too?)

If anyone is willing to try grabbing it you can use DepotDownloader to download it: https://github.com/SteamRE/DepotDownloader/releases/tag/DepotDownloader_2.4.5

Just extract that somewhere, open a command prompt window to the same folder (in file explorer, you can click on address bar and type "cmd", press enter and it'll open command prompt in the same folder), then run this command to download the German version:

`depotdownloader.exe -app 254700 -depot 254702 -username [steam username]`

(it should be safe to login with that, I've used it many times in the past without problems, and it's mentioned on PCGamingWiki for downloading older versions too, eg. https://www.pcgamingwiki.com/wiki/Talk:NieR:_Automata )

If your account has access it should begin downloading, otherwise you might get the same error message as me ("Depot 254702 (Resident Evil 4 German) is not available from this account," ;_;)

Once the download is complete you should have a depots\254702\ folder with the game files inside, the Bin32\bio4.exe file is what we're looking for, please get in contact, my email address is listed at https://github.com/emoose :) (or if you have a github account, feel free to post here!)

Many thanks to anyone that can try this for us, will help us improve re4_tweaks uncensoring patches a lot!

**Comment by linkthehylian** [comment]
I don't live in Germany, but a friend of mine had the executable you're looking for. Hope it helps.
[bio4_german.zip](https://github.com/emoose/re4-research/files/8356685/bio4_german.zip)

**Comment by emoose** [comment]
@linkthehylian oh wow awesome, was starting to think we might have lost it entirely, many thanks for finding it!

Interestingly it seems the violence-level in that EXE is set to 3 - haven't seen that before, and not really sure how that would work since all the censorship-checks only activated censorship if value was 0 (like in JP release)... for reference the uncensored release uses value 2.

Maybe GER release also changed the code for the checks as well (from the posts by WBacon on steam forums it sounds like a lot might have been changed, including data files...), will have to take a look and see if the violence-level patches I added to re4_tweaks actually work with it or not - at least it seems re4_tweaks can load in fine with that version, which is a good sign.
